scripts/setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
Was there any particular reason to use `sh` instead of `bash`?

Reply:
TBH I thought we preferred `sh` in scripts so I did this for the sake of consistency but after looking around it seems `bash` is used just as much if not more. In potential, force, pulse, aprd setup scripts we use `bash` so I am switching this to bash as its more feature rich and is used in many of our scripts.